desc/equilibrium/utils.py

```diff
@@ -181,3 +183,100 @@ def parse_axis(axis, NFP=1, sym=True, surface=None):
     else:
         raise TypeError("Got unknown axis type {}".format(axis))
     return axis
+
+
+def scale_profile(profile, inner_rho):
+    """Return a new profile whose rho=1 val is value of the old profile at inner_rho.
+
+    Args
+    ----
+        profile (Profile): Profile to scale
+        inner_rho (float): rho to take as the new LCFS rho for the new profile
+    Returns
+    -------
+        new_profile (Profile): new profile whose last value is the inner_rho value of
+        the original profile
+    """
+    params_list = []
+    modes_list = []
```

Comment:
these just list are just used in the first if. I will replace them inside the if for clarity.

Moreover, you can create them using a list compression.